spec.go

```diff
@@ -53,6 +57,8 @@ type Process struct {
 	// Cwd is the current working directory for the process and must be
 	// relative to the container's root.
 	Cwd string `json:"cwd"`
+	// ExtraFiles specifies open file descriptors that should be passed to the process.
+	ExtraFiles []os.File `json:"extraFiles"`
```

Comment:
On Fri, Aug 21, 2015 at 11:25:04AM -0700, Michael Crosby wrote:
> > +	// ExtraFiles specifies open file descriptors that should be passed to the process.
> > +	ExtraFiles []os.File `json:"extraFiles"`
> 
> @wking no

Then why not use environment variables for all runtime-specific
settings?  If there is a need to use all of:

* config.json
* runtime.json (#88)
* environment variables

how do we make decisions about sorting between them?  E.g. why use
LISTEN_FDS for passing file descriptors through and runtime.json to
pass namespace paths through?